desktop-widgets/simplewidgets.cpp

```diff
@@ -124,11 +124,6 @@ ShiftTimesDialog::ShiftTimesDialog(QWidget *parent) : QDialog(parent),
 
 void ShiftImageTimesDialog::buttonClicked(QAbstractButton *button)
 {
-	if (ui.buttonBox->buttonRole(button) == QDialogButtonBox::AcceptRole) {
-		m_amount = ui.timeEdit->time().hour() * 3600 + ui.timeEdit->time().minute() * 60;
-		if (ui.backwards->isChecked())
-			m_amount *= -1;
-	}
```

Comment:
so this leaves an empty `buttonClicked()` function? That seems... odd.

Reply:
At the risk of triggering Chesterton's Fence, I can only report that I've found no use for updating m_amount with the values from the UI when clicking the button, since any change to the values in the UI will trigger that update anyway (L254)

Reply:
I've just confirmed my suspicion. The only way to prevent `ShiftImageTimesDialog::timeEditChanged()` from triggering is by not changing the value in the `timeEdit` field _and_ having `LastImageTimeOffset=0` set in Subsurface.conf. In that case, the default value of `m_amount`, which is 0, is correct. In any other case, `timeEditChanged()` will be called after every change and `m_amount` updated accordingly. Performing this process when the Ok button is pressed serves no purpose as far as I can tell.